C++/fibonacci/fibonacci.cpp

```diff
@@ -1,16 +1,25 @@
-// Simple Program to calculate Fibonacci Sequence of an integer input
 #include <iostream>
+#include <cstdlib>
 using namespace std;
+
 int FibonacciSequence(int num) {
     if(num <= 1) {
         return num ;
     }
     return FibonacciSequence(num-1) + FibonacciSequence(num-2);
 }
-int main(){
-    cout << "Enter the Number" << endl;
-    int n ;
-    cin  >> n ;
 
-    cout << "Fibonacci Sequence term at " << n << "  " << "is " << FibonacciSequence(n) << endl;
+int main(int argc, char* argv[]){
+	int n = 10;
+	if (argc > 1) {
+		n = atoi(argv[1]);
+		if (n <= 0) {
+			n = 10;
+		} else if (n > 45) {
+			// This should be able to finish in a decent amount of time.
+			n = 45;
+		}
+	}
```

Comment:
This should compute and print Fibonacci sequence number for each index passed on command line, see Zig example. With no arguments Zig example just exits, let's be consistent

Reply:
Why just exit? That's annoying and serves no purpose. Let's make it flexible & friendly.